r/R/utilities_development.R

```diff
@@ -0,0 +1,11 @@
+#' Loads Experiment data
+#'
+#' @param exp_id
+#'
+#' @return
+#' @export
+#'
+#' @examples
+load_experiment_data <- function(exp_id = "test") {
+  return(readRDS(paste0("../data/", exp_id, "/r.rds", sep = "")))
```

Comment:
don't you have in R an equivalent of `os.path` in python that you can use here? Also, there is a second data folder under `python/src/tests`, I recommend that you name the data folder in python to be called "test_data" to avoid ambiguity